cmd/config/init/rollapp.go

```diff
@@ -49,3 +71,18 @@ func setRollappAppConfig(appConfigFilePath string, denom string) error {
 func RollappConfigDir(root string) string {
 	return filepath.Join(root, consts.ConfigDirName.Rollapp, "config")
 }
+
+func showSequencerPubKey(binary, home string) (string, error) {
```

Comment:
Isn't this func [exist](https://github.com/dymensionxyz/roller/blob/87c6104feba7c3450833dcb27c7cd32ee261ace6/cmd/register/bash_commands.go#L25)